### PR TITLE
[release] Prepare for release Linkis-1.0.0

### PR DESCRIPTION
**Pull request by sargentti**
### What is the purpose of the change
Linkis-1.0.0 includes all of Project Linkis 1.0.0.
This release mainly brings in **Orchestrator**, optimizes lots of framework code of 'Computation Governance', adds support for OnceEngineConn, which used for OLAP jobs and Streaming jobs, and simplifies the deployment of Linkis.   
Meanwhile, the /docs directory is removed, documents are moved to the [Linkis-Doc](https://github.com/WeBankFinTech/Linkis-Doc) repository.
With Orchestrator and enhanced Label Manager, Linkis 1.0.0 is able to provide multi-level labels based, cross-cluster/cross-IDC fine-grained routing, load balance, multi-tenancy, traffic control, resource control, and orchestration strategies like dual-active, hybrid-computation, etc.

### Brief change log
- Add Orchestrator
- Optimize Computation Governance framework
- Add OnceEngineConn
- Simplify deployment scripts.

### Verifying this change
This change is already covered by existing tests, such as (BVT, regression test, stress test).  

### Does this pull request potentially affect one of the following parts:
- Dependencies (does it add or upgrade a dependency): (yes)
- Anything that affects deployment: (yes)
- The MGS(Microservice Governance Services), i.e., Spring Cloud Gateway, OpenFeign, Eureka.: (no)

### Documentation
- Does this pull request introduce a new feature? (yes)
- If yes, how is the feature documented? (docs)

### CODE REVIEW

**Review by peacewong — APPROVED** [review]
LGTM.

**Review by wushengyeyouya — APPROVED** [review]
LGTM.

File: linkis-commons/linkis-common/src/main/java/com/webank/wedatasphere/linkis/common/errorcode/ErrorCodeUtils.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.webank.wedatasphere.linkis.common.errorcode;
```
Comment (Alexkun) [inline linkis-commons/linkis-common/src/main/java/com/webank/wedatasphere/linkis/common/errorcode/ErrorCodeUtils.java:1]: Need copyright there.

File: linkis-commons/linkis-common/src/main/java/com/webank/wedatasphere/linkis/common/errorcode/LinkisEngineConnErrorCodeSummary.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package com.webank.wedatasphere.linkis.common.errorcode;
+
+
+public enum LinkisEngineConnErrorCodeSummary {
+
+
+    ;
+    /**
+     * 错误码
```
Comment (Alexkun) [inline linkis-commons/linkis-common/src/main/java/com/webank/wedatasphere/linkis/common/errorcode/LinkisEngineConnErrorCodeSummary.java:9]: Please add comments in english

**Review by Alexkun — APPROVED** [review]
LGTM.

**Pull request merged**